src/pages/layer-2.js

```diff
@@ -0,0 +1,1097 @@
+            </p>
+            <p>
+              Ethereum also functions as a data availability layer for Layer 2s,
+              and if there are any disputes on previous transactions data is
+              provided from Ethereum for these disputes.
+            </p>
+          </Flex50>
+          {/* TODO: Convert these inline styles into styled components */}
+          <Flex50>
+            <p>
+              <b>Ethereum as the layer 1 includes:</b>
+            </p>
+            <OrderedList
+              listData={[
+                <p>
+                  <b>A network of node operators</b> to secure and validate the
+                  network
+                </p>,
+                <p>
+                  <b>A network of block producers</b>
+                </p>,
+                <p>
+                  <b>The blockchain</b> itself and the history of transaction
+                  data
+                </p>,
+                <p>
+                  <b>The consensus mechanism</b> for the network
+                </p>,
+              ]}
+            />
+            <p>
+              Still confused on Ethereum?{" "}
+              <Link to="/what-is-ethereum/">Learn what Ethereum is</Link>.
+            </p>
+          </Flex50>
+        </TwoColumnContent>
+      </LightGrayContent>
+
+      <PaddedContent>
+        <TwoColumnContent>
+          <FlexContainer
+            flexPercent="50"
+            style={{
+              display: "flex",
+              alignContent: "center",
+              justifyContent: "center",
+            }}
+          >
+            <GatsbyImage
+              image={getImage(data.dao)}
+              style={{ width: "100%" }}
+              objectFit="contain"
+            />
+          </FlexContainer>
+          <FlexContainer flexPercent="50">
+            <h2>Why do we need layer 2?</h2>
+            <p>
+              The three desirable properties of a blockchain are that it is
+              <b>decentralized, secure, and scalable</b>. The{" "}
+              <Link to="https://www.ledger.com/academy/what-is-the-blockchain-trilemma">
+                blockchain trilemma
+              </Link>{" "}
+              states that a simple blockchain architecture can only achieve two
+              out of three. Want a secure and decentralized blockchain? You need
+              to sacrifice scalability. This is where layer 2 networks come in.
+            </p>
+            <p>
+              Ethereum has reached the network's current capacity with{" "}
+              <Link to="https://etherscan.io/chart/tx">
+                1+ million transactions per day
+              </Link>
+              , with high demand for each of these transactions. The success of
+              Ethereum and the demand to use it has caused gas prices to rise
+              substantially. Therefore the{" "}
+              <Link to="/developers/docs/scaling/">
+                need for scaling solutions
+              </Link>{" "}
```

Comment:
Not a fan of this link.

We're linking out to a more complex page (developer doc) that is somewhat similar in content to the current page but in a less digestible way.